cargo-verify/src/main.rs

```diff
@@ -274,6 +279,9 @@ fn process_command_line() -> CVResult<Opt> {
     // Use the user specified number of jobs, or the number of CPUs.
     opt.jobs = opt.jobs_arg.unwrap_or(num_cpus::get());
 
+    // Normal mode is verbosity 1, quiet is 0
+    opt.verbose = if opt.quiet { 0 } else { opt.verbose + 1 };
```

Comment:
This should be done earlier (before the use of `info_at!` above).

Reply:
Good catch